packages/near-api-js/test/providers.test.js

```diff
@@ -222,7 +223,7 @@ test('final tx result', async() => {
     expect(nearApi.providers.getTransactionLastResult(result)).toEqual({});
 });
 
-test('final tx result with null', async() => {
+test('final tx result with null', async () => {
```

Comment:
Please update this test in `packages/accounts/test/providers.test.js` and leave this one unchanged as duplicated tests are slated for removal.